misc/make_changelog.py

```diff
@@ -32,7 +33,7 @@ def format(c):
 
     for i, c in enumerate(commits):
         s = format(c)
-        if s.startswith('[release]'):
+        if c in commits_with_tags:
```

Comment:
tag can also has names: so maybe we can name our release tag v0.x.x[rc] and check here?

Reply:
That sounds good. However, we support commands like `python3 make_changelog.py v0.7.32` to show changelog of `v0.7.32`. This requires us to maintain backward-compatibility. Unfortunately, early versions of Taichi doesn't follow this naming rule (e.g., no `v`). IMO if we only use tags for releases, this implementation without format checking will be fine.

Reply:
okay hmmm let's do this for now, would like to make it more restricted in the future ;). Since github doesn't have protected tags, our make_changelog.py witout any enforcement of "only use tags for releases".